data-runtime/build.gradle

```diff
@@ -11,6 +11,7 @@ dependencies {
 	annotationProcessor libs.micronaut.docs
 
 	api projects.dataModel
+	api mnSerde.micronaut.serde.support
```

Comment:
Can you try to remove `http` dependency that was pulling the media types conversion?

Reply:
It is used in AbstractRepositoryOperations used by other repos, but will see if it can be done.

Reply:
Removed